packages/dnb-eufemia/src/components/date-picker/DatePickerPortal.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {
+  useCallback,
+  useEffect,
+  useLayoutEffect,
+  useState,
+} from 'react'
+import ReactDOM from 'react-dom'
+import { DatePickerProps } from './DatePicker'
+
+type DatePickerPortalProps = React.HTMLProps<HTMLDivElement> & {
+  skipPortal?: DatePickerProps['skipPortal']
+  alignment?: DatePickerProps['alignPicker']
+  targetElementRef?: React.RefObject<HTMLElement>
+}
+
+export default function DatePickerPortal({
+  className,
+  skipPortal,
+  alignment,
+  targetElementRef,
+  children,
+}: DatePickerPortalProps) {
+  const [position, setPosition] = useState({})
+
+  const positionTimeout = React.useRef(null)
+
+  useEffect(() => {
+    if (targetElementRef.current) {
+      setPosition(getPosition(targetElementRef.current, alignment))
+    }
+  }, [])
+
+  const setPositionDebounce = useCallback(() => {
+    if (targetElementRef.current) {
+      setPosition(getPosition(targetElementRef.current, alignment))
+    }
+
+    clearTimeout(positionTimeout.current)
+    positionTimeout.current = setTimeout(() => {
+      if (targetElementRef.current) {
+        setPosition(getPosition(targetElementRef.current, alignment))
+      }
+    }, 200)
+  }, [alignment, targetElementRef])
+
+  useLayoutEffect(() => {
```

Comment:
```suggestion
  useEffect(() => {
```

This method is not so crucial to mount early.